index.html

```diff
@@ -39,17 +39,143 @@ <h3 class="connect">LET'S CONNECT</h3>
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <section class="about">
+        <div>
+          <div class="first-heading">
+            <h2>About Myself</h2>
+          </div>
+          <div class="description">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+              work
+              and
+              experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          </div>
+          <div>
+            <h3 class="connect">LET'S CONNECT</h3>
+            <ul class="social">
+              <li class="social"><a href="#"><img src="./images/Vector.svg" alt=""></a></li>
+              <li class="social"><a href="#"><img src="./images/Linkedin icon.svg" alt="Linkedin"></a></li>
+              <li class="social"><a href="#"><img src="./images/Vector (1).svg" alt=""></a></li>
+              <li class="social"><a href="#"><img src="./images/Vector (2).svg" alt="github"></a></li>
+              <li class="social"><a href="#"><img src="./images/Vector (3).svg" alt="twitter"></a></li>
+            </ul>
+          </div>
+          <div>
+            <button class="see-project">Get my resume</Button>
+          </div>
+          <ul class="union-list">
+            <li class="union-div">
+              <h2 class="union-header">Languages</h2>
+              <img class="union1" src="./images/Union.png" alt="">
+            <li class="about-lang">
+              <img class="union" src="./images/Ellipse 1.png" alt="">
+              <a href="http://#">JavaScript</a>
+            </li>
+            <li class="about-lang">
+              <img class="union" src="./images/Ellipse 1 (1).png" alt="">
+              <a href="http://#">HTML</a>
+            </li>
+            <li class="about-lang">
+              <img class="union" src="./images/Ellipse 1 (2).png" alt="">
+              <a href="http://#">CSS</a>
+            </li>
+          </ul>
+          </li>
+          <li class="union-div">
+            <h2>Frameworks</h2>
+            <img class="union2" src="./images/Union2.png" alt="">
+          </li>
+          <li class="union-div">
+            <h2>Skills</h2>
+            <img class="union2 space" src="./images/Union2.png" alt="">
+          </li>
+          </ul>

```

Comment:
```HTML
<ul>
  <li>
    <h2>Languages</h2>
    <ul>
       <li>
         <img src="./images/Ellipse 1.png" alt="">
         <a href="http://#">JavaScript</a>
       </li>
       <li> . . .  </li>
       <li> . . .  </li>
    </ul>
  </li>
  <li>
    <h2>Frameworks</h2>
     . . . 
  </li>
  <li> . . . </li>
</ul>
```

As suggested by the previous reviewer, please use `ul` and `li` for categories in the "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items.

Kindly note the structure as given above.
- The `<ul>` element is for grouping a collection of items
- The `<li>` HTML element is used to represent an item in a list. It must be contained in a parent element `ul` (in this case)